src/components/Input/Input.tsx

```diff
@@ -30,6 +31,9 @@ interface BaseProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 's
     label?: string;
     /** Helper text under the input, display the error when the context is critical */
     helperText?: string;
+    /**  Controls whether space is reserved for error messages under the input field when validation is expected
+     *  to avoid "jumping" UI. */
+    reserveErrorMessageSpace?: boolean;
```

Comment:
normally I think we name booleans starting with is/do. then it is easier to understand that it is not a function. I'm not sure we have it as a "hard rule". What do you think?